internal/wrappers/headers/server.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef _KUZZLE_SERVER_HPP
+#define _KUZZLE_SERVER_HPP
+
+#include "exceptions.hpp"
+#include "core.hpp"
+
+namespace kuzzleio {
+  class Server {
+    server *_server;
+    Server();
+
+    public:
+      Server(Kuzzle* kuzzle);
+      virtual ~Server();
+      std::string getAllStats(query_options* options=NULL) Kuz_Throw_KuzzleException;
+      std::string getLastStats(time_t start, time_t end, query_options* options=NULL) Kuz_Throw_KuzzleException;
```

Comment:
According to Kuzzle, API, `getLastStats` should take no parameters.
And we need another method `getStats(start, end)` :
http://docs.kuzzle.io/api-documentation/controller-server/get-last-stats/
http://docs.kuzzle.io/api-documentation/controller-server/get-stats/

Reply:
Oh it was not listed in the issue..

Reply:
yes it was missing, I noticed that while doing the javascript code.